src/ngx_http_lua_headers.c

```diff
@@ -146,9 +161,36 @@ ngx_http_lua_ngx_req_raw_header(lua_State *L)
     dd("size: %d", (int) size);
 
     if (hc->nbusy) {
+#if defined(nginx_version) && nginx_version >= 1011011
+        if (hc->nbusy > prealloc_nbusy) {
+            if (busy_bufs_ptrs != NULL)
+                ngx_pfree(ngx_cycle->pool, busy_bufs_ptrs);
```

Comment:
The nginx coding style requires `{}` to be used for `if` statement at all times.